near-contract-standards/src/non_fungible_token/token.rs

```diff
@@ -3,8 +3,10 @@ use near_sdk::serde::{Deserialize, Serialize};
 use near_sdk::AccountId;
 use std::collections::HashMap;
 
+/// Note that token IDs for NFTs are strings on NEAR. It's still fine to use autoincrementing numbers as unique IDs if desired, but they should be stringified. This is to make IDs more future-proof as chain-agnostic conventions and standards arise, and allows for more flexibility with considerations like bridging NFTs across chains, etc.
```

Comment:
This comment seems too long for one line, can mess with rust fmt or be hard to read (esp on a phone which I am now). Maybe worth splitting up?